src/VisualStudio/Core/Def/Implementation/Watson/WatsonReporter.cs

```diff
@@ -18,44 +15,20 @@ internal class WatsonReporter
         /// <param name="exception">Exception that triggered this non-fatal error</param>
         public static void Report(Exception exception)
         {
-            // Log the exception regardless of whether the Watson will be throttled.
-            Logger.Log(FunctionId.NonFatalWatson, KeyValueLogMessage.Create(dict =>
-            {
-                dict["ExceptionType"] = exception.GetType().FullName;
-                dict["ExceptionStack"] = GetStackTraceString(exception);
-            }));
-
-            using (var report = WatsonErrorReport.CreateNonFatalReport(new ExceptionInfo(exception, "Roslyn")))
-            {
-                // Ignore the return value on purpose, we don't care if it actually gets submitted
-                report.ReportIfNecessary();
-            }
+            Report("Roslyn NonFatal Watson", exception);
         }
 
-        /// <remarks>
-        /// <see cref="Exception.StackTrace"/> produces inconvenient strings: 
-        /// there are line breaks and "at" is localized.
-        /// </remarks>
-        private static string GetStackTraceString(Exception e)
+        /// <summary>
+        /// Report Non-Fatal Watson
+        /// </summary>
+        /// <param name="description">watson description</param>
```

Comment:
Can this document what the guidance is for this string? This isn't terribly useful otherwise.